proxy/src/bind.rs

```diff
@@ -217,16 +229,20 @@ where
         Reconnect::new(proxy)
     }
 
-    pub fn new_binding(&self, ep: &Endpoint, protocol: &Protocol) -> Binding<B> {
-        if protocol.can_reuse_clients() {
+    pub fn new_bound_service(&self, ep: &Endpoint, protocol: &Protocol) -> BoundService<B> {
```

Comment:
TIOLI: this method was called `bind_service` before being renamed to `new_bound_service`, perhaps just change it back?